packages/react-router-dom/__tests__/DataBrowserRouter-test.tsx

```diff
@@ -12,25 +12,27 @@ import {
   prettyDOM,
 } from "@testing-library/react";
 import "@testing-library/jest-dom";
+import {
+  useNavigate,
+  UNSAFE_DataRouterStateContext,
+  UNSAFE_resetModuleScope,
+} from "react-router";
```

Comment:
I think we need to think through the unit testing approach in here a bit more.  The module resolution here is now non-deterministic depending on whether a `dist/` directory exists which is unfortunate.  It seems `jest` first looks for the file pointed to by `packageJson.main` and if it doesn't find that, it ends up falling back on the `index.ts` file.  This wasn't a problem previously because the built files never matched...

So - if you have a `dist/` directory, you are getting those (likely stale) files and aren't getting realtime cross-package changes with `jest --watch` (and maybe not even same-package) if you import via the package name :(

Will need to do some digging and see if there's a way around that, or if we should change all our unit tests imports to come from `../index` and the like.

Reply:
Fixed with a new `jest` `moduleNameMappers` config